### PR TITLE
AA changes for date select input

### PR DESCRIPTION
**Pull request by kialj876**
*Issue:*https://github.com/bcgov/entity/issues/19440

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-116-9oa8xp9k.web.app

File: btr-web/btr-common-components/utils/date.ts [hunk 1]
```diff
@@ -6,6 +6,8 @@ import moment from 'moment'
 export function dateStringToDate (dateString: string) {
   // convert to date
   const date = new Date(dateString)
+  // @ts-ignore
+  if (isNaN(date)) { return null }
```
Comment (hfekete) [inline btr-web/btr-common-components/utils/date.ts:10]: are you sure that this will work ?
Is JS treating date as number ?

**Review by hfekete — APPROVED** [review]
If the isNaN function on date will work, then I approve. :D
Everything else looks good!

File: btr-web/btr-common-components/utils/date.ts (hunk 1 above)
Comment (kialj876) [reply btr-web/btr-common-components/utils/date.ts:10]: yeah I read about this online and then tested it -- catches invalid dates (i.e. if I called this function with 'qwe')

**Pull request merged**